### DISCUSSION TITLE
How to disable popup during startup

### DISCUSSION THREAD

**Discussion by OpenFoam-User** [LazyVim/LazyVim#1479] [Q&A] [answered]
Hi, I am a new user of Lazyvim.

Whenever I open a file using LazyVim, I get this popup on the right:

![image](https://github.com/LazyVim/LazyVim/assets/67183566/4cf86a0e-aa80-4884-aaca-430d4011fd44)

How can I get rid of it? I find it annoying and distracting.

Thank you.

**Comment by dpetka2001** [comment, marked as answer]
If you update the plugins you won't get the notification. Other than that you can look at `lazy.nvim`'s default [configuration](https://github.com/folke/lazy.nvim#%EF%B8%8F-configuration) and specifically the option `checker.notify` which is set to `true` by default. If you set it to `false` you won't get any notifications about updates.

**Reply by dpetka2001** [reply]
`lazy.nvim` is the package manager LazyVim uses and that pop up message you're showing comes from the package manager to notify you that there are available packages for updating.

**Comment by Emile-Durkheim** [comment]
Instead of disabling the checker completely, in `~/.config/nvim/lua/config/lazy.lua`, you can also:

- Disable the notifications, but not the checker: `checker = { enabled = true, notify = false }`
- Reduce the frequency of checking. In this case I set it to 24h in seconds. `checker = { enabled = true, frequency = 86400 }`

**Reply by OpenFoam-User** [reply]
Thank you

**Reply by 123abch** [reply]
That's a very detailed answer, thanks!
